mandatory/3-function-output.js

```diff
@@ -1,14 +1,17 @@
 // Add comments to explain what this function does. You're meant to use Google!
+// return a radom number between 1 and 10
 function getNumber() {
   return Math.random() * 10;
 }
 
 // Add comments to explain what this function does. You're meant to use Google!
+// join two arguments 
 function s(w1, w2) {
   return w1.concat(w2);
 }
 
 function concatenate(firstWord, secondWord, thirdWord) {
+  return firstWord.concat(" "+secondWord + " ", thirdWord);
```

Comment:
Hey great work just some refactoring tip here...

```diff
- return firstWord.concar(" "+secondWord + " ", thirdWord);
+ return `${firstWord} ${secondWord}, ${thirdWord}`;
```

- So your solution there works but prone to errors that can be caused **by mistake you can miss on spacing your quotation marks then the words get attached together as one word which is something you did not intend to do** 
- So template literals tend to be the most effective in concatenating strings etc